app/views/likes/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'form'%>
+
```

Comment:
- [ ] Great job so far in your implementation of forms in your blog app. However, I noticed some adjustments that would help improve your app's functionality and user experience.

  - I think it would be better if you don't create a view for your likes because you don't need it, typically on any website or app that has a like functionality you mostly see that users just click on the like button on a post, or photo to have a like added. However, in your case, you have a dedicated `route` and `view` that contains just the like button which seems a bit confusing therefore I would recommend that you get rid of the like `new` action and view, and in the post details page implement the like functionality such that user will have to just click on the like button to like the post. In the attached screenshot is your like view:
  
<img width="1159" alt="Screenshot 2024-01-20 at 21 07 05" src="https://github.com/AlierPM/BlogPress/assets/107192076/2f12ce27-f1c8-42ec-9cf3-d99a9403b1f6">

Reply:
@Olamarx  and @Strangeal : Since we are focusing on functionality, UX or UI is the NOT a main requirement now, as per this project, I  hope you understand my case. thanks for the suggestions though.